.circleci/config.yml

```diff
@@ -160,6 +160,8 @@ jobs:
           source .circleci/define_build_environment_variables
           echo "export FALA_IMAGE=$ECR_DEPLOY_IMAGE" >> $BASH_ENV
           echo "export A11Y_ENABLED=true" >> $BASH_ENV
+          echo "export FALA_TESTS_ONLY=true" >> $BASH_ENV
+          echo "export E2E_BRANCH=EL-1440-add-conditional-fala-tests" >> $BASH_ENV
```

Comment:
You've left `"export E2E_BRANCH=EL-1440-add-conditional-fala-tests" >> $BASH_ENV` In your PR.

Reply:
without it I can't prove that the tests run properly. I guess this should be in a separate branch or something...